packages/cli/src/commands/git/connect.ts

```diff
@@ -64,20 +65,37 @@ export default async function connect(
     );
     return 1;
   }
-  const remoteUrl = pluckRemoteUrl(gitConfig);
-  if (!remoteUrl) {
+  const remoteUrls = pluckRemoteUrls(gitConfig);
+  if (!remoteUrls) {
     output.error(
-      `No remote origin URL found in your Git config. Make sure you've configured a remote repo in your local Git config. Run ${chalk.cyan(
+      `No remote URLs found in your Git config. Make sure you've configured a remote repo in your local Git config. Run ${chalk.cyan(
         '`git remote --help`'
       )} for more details.`
     );
     return 1;
   }
-  output.log(`Identified Git remote "origin": ${link(remoteUrl)}`);
+
+  let remoteUrl: string;
+
+  if (Object.keys(remoteUrls).length > 1) {
+    output.log(`Found multiple remote URLs.`);
+    remoteUrl = await listRemoteUrls(client, remoteUrls);
```

Comment:
I think `listRemoteUrls` would make more sense as something like `selectRemoteUrl`. That better describes what's happening from the call site's perspective.